.github/workflows/ci.yml

```diff
@@ -39,7 +39,9 @@ jobs:
         shell: bash
         run: |
           python3 -m pip install -e .[test]
-      - name: Test
-        shell: bash
-        run: |
-          scripts/test
+      - name: Test with Pytest
+        run: pytest --cov=snitun --cov-report=xml
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
This should not be done in a matrix, do it in the lowest tracked version only.

Reply:
Yeah, or alternatively store the coverage files and upload them all to codecov in one go in the coverage job.